active/0002-centalized-mgmt-resource-types-provider-versions.md

```diff
@@ -0,0 +1,262 @@
+---
+REP Number: 0002
+Author: asilverman (Ariel Silverman)
+Start Date: 2023-12-19
+Feature Status: Private Preview
+Bicep Issue Number(s): [12202](https://github.com/Azure/bicep/issues/12202)
+---
+
+# Title - Centralized management of resource type provider versions using package.json
```

Comment:
nit - this still says `package.json` but I think you mean `bicepconfig.json`?

Reply:
yes, thank you. Fixed now